jerry-debugger/jerry-client.py

```diff
@@ -16,12 +16,23 @@
 
 import socket
 import sys
+from struct import *
+
+# Define the debugger buffer types
+JERRY_DEBUGGER_BREAKPOINT_LIST = 1
+JERRY_DEBUGGER_BREAKPOINT_LIST_END = 2
+JERRY_DEBUGGER_FUNCTION_NAME = 3
+JERRY_DEBUGGER_FUNCTION_NAME_END = 4
+JERRY_DEBUGGER_SOURCE_FILE_NAME = 5
+JERRY_DEBUGGER_SOURCE_FILE_NAME_END = 6
+JERRY_DEBUGGER_UNIQUE_START_BYTE_CODE_CPTR = 7
 
 PORT = 5001
+MAX_BUFFER_SIZE = 64  # Need to be the same as the jerry debugger MAX_BUFFER_SIZE
```

Comment:
I think the point is we don't need to know this size. Only the minimum should be defined.